### DISCUSSION TITLE
Possible markdown support?

### DISCUSSION THREAD

**Discussion by luohao123** [flyerhq/flutter_chat_ui#154] [Ideas & Requests] [answered]
Just wonder if there any option to make users enable markdown support.

I noticed that there already have link support, but bold and ordered check list does't support

**Comment by demchenkoalex** [comment, marked as answer]
Yeah, this would be a nice feature. Let me create a good first issue from this.

**Reply by demchenkoalex** [reply]
Created issue, so resolving this one.
